types/mongodb/index.d.ts

```diff
@@ -1356,6 +1367,25 @@ export interface ReplaceOneOptions extends CommonOptions {
     bypassDocumentValidation?: boolean;
 }
 
+/** http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#updateOne */
+export interface UpdateOneOptions extends CommonOptions {
```

Comment:
You can extend `ReplaceOneOptions` adding only `arrayFilters`, to avoid the duplication